### ISSUE TITLE
How can i save my Model?

### ISSUE THREAD

**Issue by nikkisingh111333** [chimera0/accel-brain-code#25]
hello I dont Know How Can I save My PyGan Model ..I cant find any code for this on Doc provided please do reply..

**Comment by accel-brain** [comment]
You can save the parameters with the `save_parameters` method. The type of model is a [mxnet.gluon.HybridBlock](https://mxnet.apache.org/versions/1.7/api/python/docs/api/gluon/hybrid_block.html).

**Comment by nikkisingh111333** [comment]
can you please share a piece of code how to do it and one more thing
How can i get one only after returning predicted image..i m getting 9 images while predicting pygan model..

**Comment by accel-brain** [comment]
The documentation about mxnet.gluon.HybridBlock will give you a code sample. I don't know what code you wrote at all, but I'm sure the type of your model is mxnet.gluon.HybridBlock. This is because Pygan's model inherits from HybridBlock and is implemented, as already mentioned.

**Comment by nikkisingh111333** [comment]
This is my code After successfully training PyGan on 1000 Airplane images..basically i m returning predicted image

**from keras.models import load_model
from keras.models import load_model
import matplotlib.image as mpimg
from keras.preprocessing.image import load_img
from keras.preprocessing.image import img_to_array
from keras.applications.vgg16 import preprocess_input
from keras.applications.vgg16 import decode_predictions
import matplotlib.pyplot as plt
import numpy as np
from PIL import Image
import numpy as np
#print(gan_image_generator.GAN.posterior_logs_arr)
arr = gan_image_generator.GAN.generative_model.draw()
arr=arr
print(arr[0].dtype)
arr=(arr*255).astype('uint8')
im=arr.reshape(1,131,131,3)
im=np.squeeze(im)**

the `arr` variable is returning predicted images of grid 3x3 small images of Airplane:heres the image of output shown below
![Screenshot (10)](https://user-images.githubusercontent.com/49682940/139431588-89741910-d007-44c9-aa10-6c2f4c0f84c9.png)


**what i want**
just single image of airplane generated..not this grid like object ..how can i extract one..
please give some piece of code for it..

**Comment by chimera0** [comment]
@nikkisingh111333 Please close this issue when you're done talking about how to save learned parameters.

> images of grid 3x3 small images

This depends on the structure of the image you have prepared and the processing you have implemented, such as reshape. I don't know anything about keras, but if you're using that library, maybe you're having trouble using it?

**Comment by nikkisingh111333** [comment]
ok i will open new issue for this thread.i dont understand what you are trying to saying..
